diff -uNrp a/mysql-connector-c-6.0.2/CMakeLists.txt b/mysql-connector-c-6.0.2/CMakeLists.txt
--- a/mysql-connector-c-6.0.2/CMakeLists.txt	2009-08-07 09:31:12.000000000 -0300
+++ b/mysql-connector-c-6.0.2/CMakeLists.txt	2012-08-31 23:47:34.361180933 -0300
@@ -37,7 +37,7 @@ ENDIF(CMAKE_COMPILER_IS_GNUCXX)
 
 
 SET(PROTOCOL_VERSION "10")
-SET(SHARED_LIB_VERSION "16.0.0")
+#SET(SHARED_LIB_VERSION "16.0.0")
 
 # This roughly corresponds to the MySQL server version we work with
 SET(CONNECTOR_C_MAJOR "6")
diff -uNrp a/mysql-connector-c-6.0.2/libmysql/CMakeLists.txt b/mysql-connector-c-6.0.2/libmysql/CMakeLists.txt
--- a/mysql-connector-c-6.0.2/libmysql/CMakeLists.txt	2009-08-07 09:31:12.000000000 -0300
+++ b/mysql-connector-c-6.0.2/libmysql/CMakeLists.txt	2012-08-31 23:55:57.596143745 -0300
@@ -96,10 +96,10 @@ TARGET_LINK_LIBRARIES(mysqlclient ${ZLIB
 
 ADD_LIBRARY(libmysql          SHARED ${CLIENT_SOURCES} libmysql.def)
 TARGET_LINK_LIBRARIES(libmysql ${CMAKE_THREAD_LIBS_INIT})
-STRING(REGEX REPLACE "\\..+" "" LIBMYSQL_SOVERSION ${SHARED_LIB_VERSION})
-SET_TARGET_PROPERTIES(libmysql
-                      PROPERTIES VERSION ${SHARED_LIB_VERSION}
-                                 SOVERSION ${LIBMYSQL_SOVERSION})
+#STRING(REGEX REPLACE "\\..+" "" LIBMYSQL_SOVERSION ${SHARED_LIB_VERSION})
+#SET_TARGET_PROPERTIES(libmysql
+#                      PROPERTIES VERSION ${SHARED_LIB_VERSION}
+#                                 SOVERSION ${LIBMYSQL_SOVERSION})
 IF(OPENSSL_LIBRARIES)
   TARGET_LINK_LIBRARIES(libmysql ${OPENSSL_LIBRARIES} ${OPENSSL_LIBCRYPTO})
 ENDIF(OPENSSL_LIBRARIES)
@@ -111,21 +111,21 @@ IF(MATH_LIBRARY)
   TARGET_LINK_LIBRARIES(libmysql ${MATH_LIBRARY})
 ENDIF(MATH_LIBRARY)
 
-IF(NOT WIN32)
-  ADD_CUSTOM_COMMAND(OUTPUT "libmysqlclient${CMAKE_SHARED_LIBRARY_SUFFIX}"
-                            "libmysqlclient_r${CMAKE_SHARED_LIBRARY_SUFFIX}"
-                    COMMAND ${CMAKE_COMMAND} ARGS -E remove -f libmysqlclient${CMAKE_SHARED_LIBRARY_SUFFIX}
-                    COMMAND ${CMAKE_COMMAND} ARGS -E create_symlink libmysql${CMAKE_SHARED_LIBRARY_SUFFIX} libmysqlclient${CMAKE_SHARED_LIBRARY_SUFFIX}
-                    COMMAND ${CMAKE_COMMAND} ARGS -E remove -f libmysqlclient_r${CMAKE_SHARED_LIBRARY_SUFFIX}
-                    COMMAND ${CMAKE_COMMAND} ARGS -E create_symlink libmysql${CMAKE_SHARED_LIBRARY_SUFFIX} libmysqlclient_r${CMAKE_SHARED_LIBRARY_SUFFIX}
-                    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/libmysql
-                    DEPENDS libmysql)
-
-  ADD_CUSTOM_TARGET(LIBMYSQL_SYMLINKS
-                    ALL
-                    DEPENDS "libmysqlclient${CMAKE_SHARED_LIBRARY_SUFFIX}"
-                            "libmysqlclient_r${CMAKE_SHARED_LIBRARY_SUFFIX}")
-ENDIF(NOT WIN32)
+# IF(NOT WIN32)
+#   ADD_CUSTOM_COMMAND(OUTPUT "libmysqlclient${CMAKE_SHARED_LIBRARY_SUFFIX}"
+#                             "libmysqlclient_r${CMAKE_SHARED_LIBRARY_SUFFIX}"
+#                     COMMAND ${CMAKE_COMMAND} ARGS -E remove -f libmysqlclient${CMAKE_SHARED_LIBRARY_SUFFIX}
+#                     COMMAND ${CMAKE_COMMAND} ARGS -E create_symlink libmysql${CMAKE_SHARED_LIBRARY_SUFFIX} libmysqlclient${CMAKE_SHARED_LIBRARY_SUFFIX}
+#                     COMMAND ${CMAKE_COMMAND} ARGS -E remove -f libmysqlclient_r${CMAKE_SHARED_LIBRARY_SUFFIX}
+#                     COMMAND ${CMAKE_COMMAND} ARGS -E create_symlink libmysql${CMAKE_SHARED_LIBRARY_SUFFIX} libmysqlclient_r${CMAKE_SHARED_LIBRARY_SUFFIX}
+#                     WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/libmysql
+#                     DEPENDS libmysql)
+#
+#   ADD_CUSTOM_TARGET(LIBMYSQL_SYMLINKS
+#                     ALL
+#                     DEPENDS "libmysqlclient${CMAKE_SHARED_LIBRARY_SUFFIX}"
+#                             "libmysqlclient_r${CMAKE_SHARED_LIBRARY_SUFFIX}")
+# ENDIF(NOT WIN32)
 
 IF(NOT WIN32)
   # Avoid a redundant 'lib'
@@ -148,7 +148,7 @@ ELSE(WIN32)
   INSTALL(TARGETS libmysql mysqlclient
           LIBRARY DESTINATION "lib"
           ARCHIVE DESTINATION "lib")
-  INSTALL(FILES ${CMAKE_SOURCE_DIR}/libmysql/libmysqlclient${CMAKE_SHARED_LIBRARY_SUFFIX}
-                ${CMAKE_SOURCE_DIR}/libmysql/libmysqlclient_r${CMAKE_SHARED_LIBRARY_SUFFIX}
-          DESTINATION "lib")
+#  INSTALL(FILES ${CMAKE_SOURCE_DIR}/libmysql/libmysqlclient${CMAKE_SHARED_LIBRARY_SUFFIX}
+#                ${CMAKE_SOURCE_DIR}/libmysql/libmysqlclient_r${CMAKE_SHARED_LIBRARY_SUFFIX}
+#          DESTINATION "lib")
 ENDIF(WIN32)
